fixtures/sample-app/pom.xml

```diff
@@ -38,6 +51,13 @@
         <app.stage.dockerDirectory>src/main/docker-openjdk</app.stage.dockerDirectory>
```

Comment:
With these changes, the `maven-resources-plugin` will copy the correct dockerfile for the selected profile into `target/docker` right? If so, can we delete this property?